src/dotnet-install.ps1

```diff
@@ -275,6 +294,19 @@ function Get-NormalizedProduct([string]$Runtime) {
     }
 }
 
+function Get-NormalizedOs([string]$Os) {
+    Say-Invocation $MyInvocation
+
+    # REVIEW: I know I am missing a lot here.
```

Comment:
I am aware that I am missing a lot of the logic for different systems here, it should be simple porting it from the .sh script, but I want to first know if you are willing to take this change before committing deeper.